time_tracker/core/usecases.py

```diff
@@ -12,17 +12,14 @@ def get_entries_list(data_source, repo_name, committer=None, issue=None,
     if issue:
         issue = repo.get_issue(issue)
         entries.extend(issue.get_entries())
-    else:
-        for issue in repo.get_issues():
-            entries.extend(issue.get_entries())
-
     if committer:
```

Comment:
Filtering repo by committer should return all commits of this user AND all comments authored by this user from all issues.